app/views/internal/configs/show.html.erb

```diff
@@ -181,6 +181,49 @@
           </div>
         </div>
 
+        <div class="card mt-3">
+          <%= render partial: "card_header",
+                     locals: {
+                       header: "Credits",
+                       state: "collapse",
+                       target: "creditsBodyContainer",
+                       expanded: "false"
+                     } %>
+
+          <div id="creditsBodyContainer" class="card-body collapse hide" aria-labelledby="creditsBodyContainer">
```

Comment:
I think this is good enough for now. The reason I explicitly list all 4 options is that I wanted to give some more context on each. Once we make the limits themselves configurable we can revisit this.